### PR TITLE
Wording Fixes for TW Localization

### PR DESCRIPTION
**Pull request by the3dsandwich**
Some of the text needs reviewing. hopefully this time around we can catch all the errors and weird word choices I've made.

### CODE REVIEW

File: app/src/main/res/values-b+zh+TW/localized.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <string name="auto_skill_item_open_maker">產生器</string>
     <string name="auto_skill_support_any">任何</string>
 
-    <string name="support_imgs_extracted">成功擷取支援從者圖片</string>
+    <string name="support_imgs_extracted">成功截取支援從者圖片</string>
```
Comment (reconman) [inline app/src/main/res/values-b+zh+TW/localized.xml:15]: Is 截取支 really correct? Online translation tools translate it to to English as `interception`. Are words like 提取 or 解压 better? What is done here is that the images are copied from the APK to the file system.

Comment (the3dsandwich) [reply app/src/main/res/values-b+zh+TW/localized.xml:15]: I see. Then how about using just "copy" `複製` for it?
It's different from the support image maker, right?

Comment (the3dsandwich) [reply app/src/main/res/values-b+zh+TW/localized.xml:15]: I'll also change `<string name="support_menu_extract_default_support_images">截取預設支援從者圖片</string>` as well

Comment (MathewSachin) [reply app/src/main/res/values-b+zh+TW/localized.xml:15]: it's different from support image maker.
Only pictures are extracted.
The default images contained in the app for Merlin, Waver, Skadi, etc.

**Review by the3dsandwich — COMMENTED** [review]
`support_imgs_extracted` changed to (rough translation) `image copy success` (圖片複製成功)
`support_menu_extract_default_support_images` changed to (rough translation) `copy default support image` (複製預設支援圖片)

As app is copying images I think using "copy" may be better to reflect the result

**Comment by the3dsandwich** [comment]
It would be nice if we can have another TW user take a look at the translation. To be honest I'll still be using the English interface even after we fix all the text problem.

By the way, when is Gilfest for NA?

**Comment by reconman** [comment]
> It would be nice if we can have another TW user take a look at the translation.

You're the only TW user of FGA, I think. You would have to ask around in taiwanese FGO forums.

> By the way, when is Gilfest for NA?

Around September 20th.

**Comment by MathewSachin** [comment]
> Around September 20th.

Does that mean we get a dead week?

**Comment by reconman** [comment]
> > Around September 20th.
> 
> Does that mean we get a dead week?

Yes, probably. But I just noticed that Fate/Zero started 5 days early, so Gilfest may also start early. Or they give us a few days of break.

**Comment by the3dsandwich** [comment]
Well then. If there happens to be any FGO TW user around and found the translation text unacceptable, then I suppose we can fix it then. Somehow TW forums are quite strict about assisting tools, so that's sad.

TW will have Christmas from Sep 11th. I guess I'll be doing my best to record videos (when AP won't be a problem) and seeing if anything need tweaking.

**Comment by the3dsandwich** [comment]
On this note, just realized I have the CEs for Gilfest still lying around. I can help add some of them in if you don't mind

**Comment by MathewSachin** [comment]
NA doesn't need to make images of event CEs anymore. We already got the JP feature of filtering by event CE

**Comment by reconman** [comment]
> TW will have Christmas from Sep 11th.

Are all seasonal events not at the time they should be? 

> NA doesn't need to make images of event CEs anymore. We already got the JP feature of filtering by event CE

Which doesn't fully work in this event, for some reason.

**Comment by MathewSachin** [comment]
> Which doesn't fully work in this event, for some reason.

Fate/Zero? Maybe you have the Bonus servant setting ON.

**Comment by MathewSachin** [comment]
And other CEs were in the general pool. So, they're not really event CEs

**Comment by the3dsandwich** [comment]
> Are all seasonal events not at the time they should be?

I don't know about other servers, but we on TW do Christmas in summer and have dead weeks that lasts 3 months.

> NA doesn't need to make images of event CEs anymore. We already got the JP feature of filtering by event CE

Oh, right. Almost totally forgot about this feature. So I guess I don't need to help add that (?) Cuz I noticed the CE for summer event is still there so thought it's still needed

**Comment by the3dsandwich** [comment]
(Gilfest2020) CKT
(Gilfest2020) Return Match
(Gilfest2020) Shining Skyscrapers

I'll put them here (the ones I have) just in case

**Comment by reconman** [comment]
@MathewSachin can we remove https://github.com/MathewSachin/Fate-Grand-Automata/blob/master/app/src/main/assets/Support/ce/Painting%20Summer.png?

**Comment by MathewSachin** [comment]
I added Painting Summer because it is one of those 50% starting gauge general purpose CEs.
We don't bundle event CEs in the app otherwise.

People can use Support image maker is they really need those images.

**Comment by the3dsandwich** [comment]
Ah I see. No problem with CEs then

**Pull request merged**